modules/graceful-shutdown.adoc

```diff
@@ -42,7 +42,7 @@ $ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-s
 +
 [source,terminal]
 ----
-$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1 <1>; done
+$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1 ; done <1>
```

Comment:
@xenolinux hi! Thank you for addressing this issue.

You could change `1 ;` to `1;`. Removing the space before the semi-colon is consistent with the example earlier in the command. Thanks!

Reply:
@pneedle-rh Addressed